README.md

```diff
@@ -103,27 +99,52 @@ The SDK cannot add this parameter itself because it part of the signed SSO URL.
 
 ## The Auth Endpoint
 
-Because the embed secret needs to be carefully guarded, embed SSO URLs cannot be created in the browser. To make the process easier and secure, you can instead do the following:
+In order to use the embed SDK on the frontend you must supply a backend service that handles authentication. This service is called by the SDK to generate a signed iframe URL that is unique to the requesting user. The best practice is for your backend service to generate the URL by calling the Looker API. 
```

Comment:
Is it best practice? It increases latency and changes security considerations by requiring a Looker service account API credential. Why would we not portray both options (signing and API) as equally valid?